File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/Maquina.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class Maquina {

    private String id_maquina;
    private String nome;
    private String senha;
    private String fk_empresa;

    public Maquina(String id_maquina, String nome, String senha,
            String fk_empresa) {
        this.id_maquina = id_maquina;
        this.nome = nome;
        this.senha = senha;
        this.fk_empresa = fk_empresa;
    }
}

File: web/models/maquina.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    let maquina = sequelize.define('maquina',{	
		id: {
			field: 'id_maquina',
			type: DataTypes.INTEGER,
			primaryKey: true,
			autoIncrement: false
		},	
		nome: {
			field: 'nome',
			type: DataTypes.STRING,
			allowNull: false
		},
		senha: {
			field: 'senha',
			type: DataTypes.STRING, 
			allowNull: false
		},
        fk_empresa: {
			field: 'fk_empresa',
			type: DataTypes.STRING, 
			allowNull: false
		},
	}, 
	{
		tableName: 'maquina', 
		freezeTableName: true, 
		underscored: true,
		timestamps: false,
	});

    return maquina;
};
